src/ChatMessage.ts

```diff
@@ -9,6 +9,7 @@ export type ChatMessageParams<T extends MessageContentType> = {
   senderId: string;
   direction: MessageDirection;
   content: MessageContent<T>;
+  date: Date;
```

Comment:
@Anberm 
Please remove this commit from the PR.
PRs should contain only changes for one specified thing. This is to keep the code history clean.

Also, I think we can add the new property, but it should be named as intended.
Is it send date, receive date, received by server date, or read date?

If you want it to be merged, please change the property name and create separate PR.
Thank you in advance :)